backend/database_handler/models.py

```diff
@@ -91,7 +91,7 @@ class TransactionsAudit(Base):
     __table_args__ = (PrimaryKeyConstraint("id", name="transactions_audit_pkey"),)
 
     id: Mapped[int] = mapped_column(Integer, primary_key=True, init=False)
-    transaction_id: Mapped[Optional[int]] = mapped_column(Integer)
+    transaction_hash: Mapped[Optional[str]] = mapped_column(String(255))
```

Comment:
nit: could you add this as a Foreign Key?

Reply:
Added foreign key constraint, please review